dev-libs/libpreludedb/files/libpreludedb-5.1.0-fix_gtkdoc_1.32.patch

```diff
@@ -0,0 +1,13 @@
+--- a/docs/api/libpreludedb-docs.xml	2019-09-17 18:49:45.721227852 -0400
++++ b/docs/api/libpreludedb-docs.xml2	2019-09-30 04:06:36.973341933 -0400
```

Comment:
You don't really need the timestamps here, it doesn't add any value. Please remove them.

Reply:
done